extensions/wikia/WallNotifications/WallNotificationsEveryone.class.php

```diff
@@ -167,28 +216,22 @@ public function getEntityProcessed( $userId, $entityKey ) {
 			__METHOD__
 		);
 
-		if ( $row->cnt == 0 ) {
-			wfProfileOut( __METHOD__ );
-			return false;
-		}
-
-		$this->setEntityProcessed( $userId, $entityKey );
 		wfProfileOut( __METHOD__ );
-		return true;
+		return ( $row->cnt == 0 ) ? false : true;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "true" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)